### PR TITLE
gyb on Windows

### PR DESCRIPTION
**Pull request by compnerd**
Closes #14280

### CODE REVIEW

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by compnerd** [comment]
CC: @AndrewSB

**Comment by AndrewSB** [comment]
Closes #14280

**Comment by AndrewSB** [comment]
@compnerd this looks good, edit your first comment to include "Closes #14280"

**Comment by compnerd** [comment]
@swift-ci please test

File: include/swift/Syntax/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
-set(line_directive "#line" "%(line)d" "\"%(file)s\"")
-set(SWIFT_GYB_FLAGS
-  --line-directive "'${line_directive}'")
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL Windows)
+  set(SWIFT_GYB_FLAGS --line-directive "^\"#line %(line)d \\\"%(file)s\\\"^\"")
+else()
+  set(SWIFT_GYB_FLAGS --line-directive "\'#line" "%(line)d" "\"%(file)s\"\'")
```
Comment (AndrewSB) [inline include/swift/Syntax/CMakeLists.txt]: Ahh, I see. Just single escapes 👍

**Review by AndrewSB — APPROVED** [review]
🙌

How did the CI succeed? The last comment from @swift-ci indicated a failure

**Pull request merged**